payload-examples/api.github.com/package/published.npm.payload.json

```diff
@@ -3,7 +3,7 @@
   "package": {
     "id": 10696,
     "name": "hello-world-npm",
-    "namespace": "Codertocaat/hello-world-npm",
+    "namespace": "Codertocat/hello-world-npm",
```

Comment:
This feels like a typo -- at least I can't find it in npm ...

Reply:
It's just an example payload. The package doesn't necessarily exist on npm